frontend/src/hooks/useRouterFilter.ts

```diff
@@ -51,8 +51,8 @@ export const useRouterFilter = <T extends BasePropertyFilter>(
   React.useEffect(() => {
     const filterParams = new URLSearchParams(filter as any);
     const allParams = {
-      ...queryString.parse(filterParams.toString()),
       ...queryString.parse(history.location.search),
+      ...queryString.parse(filterParams.toString()),
```

Comment:
Is there a reason for the order change?

Reply:
it started applying them values to the url properly afterward... not exactly sure the reason why but it works now. (ex) in dev right now if you apply a filter the URL will not change

Reply:
the old order would overwrite the new params with any existing values, I must have messed this up at some point.